server/api/controllers/locker-signout-controller.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from "express";
+import { Logger } from "../../util/logger";
+// import { ReadWriteAPIController } from "./read-write-api-controller";
+// import { ReadWriteController } from "./read-write-controller";
+import LockerData from "../../data/_hidden/locker-list.json";
+import { Locker } from "../../types/locker";
+
+/* Assuming locker-list.json will have the following structure: 
```

Comment:
Followup to this thought - we should also consider what level of access a user needs to use these endpoints. For data privacy reasons, we shouldn't allow normal students to see who is assigned to lockers, nor what locker a student is assigned to. Look into the `AdminMiddleware` function when adding your endpoints - this will prevent non-admin users from using your services.

(If we want to display the user their own locker information, we can set up an endpoint to do that where we search for the logged-in user's id.  This would circumvent data privacy issues but still allow the user to get info!).